README.md

```diff
@@ -51,14 +55,22 @@ SELECTORS:
         If used in conjunction with --vm-id, then will be ignored
         TODO: handle multiple results. Currently fails in such case.
 
+  --sr-id <id>
+        XO's SR ID where to import the VM into.
```

Comment:
Can you add that SR stand for Storage Repository ?
I'm not used to Xen, so it may be useless.